index.html

```diff
@@ -3,21 +3,240 @@
+                <a href="#">English</a>
             </li>
-            <li></li>
-            <li></li>
         </ul>
     </nav>
+    <nav class="navbar navbar-expand-sm hbground">
+        <div class="container">
+            <a class="navbar-brand d-none d-md-flex" href="./index.html"><img class="header-img" src="img/work.png" alt="logo"><p>Kadoma <br> Community Development</p></a>
+            <div class="hamburger d-lg-none">
+                <img class="ham" src="img/ham2.png" alt="hamburger-menui">
+            </div>
+            <div class="d-none d-lg-block justify-content-end" id="navbarNav">
+                <div class="d-flex align-items-center justify-content-around">
+                    <a class="navbar-brand d-lg-none " href="./index.html"><img class="cont" src="img/work.png" alt="brand"><p>Kadoma <br> Community Development</p></a>
+                    <div class="close d-none">
+                        <i class="material-icons justify-content-end d-flex mx-3">close</i>
+
+                    </div>
+
+                </div>
+                <ul class="navbar-nav align-items-center">
+                    <li class="nav-item mx-2">
+                        <a class="nav-link" href="./index.html">Home</a>
+                    </li>
+                    <li class="nav-item mx-2">
+                        <a class="nav-link" href="./about.html">About</a>
+                    </li>
+                    <li class="nav-item mx-2">
+                        <a class="nav-link" href="#">Program</a>
+                    </li>
+                    <li class="nav-item mx-2">
+                        <a class="nav-link" href="#">Join</a>
+                    </li>
+                    <li class="nav-item mx-2">
+                        <a class="nav-link" href="#">Spoinsor</a>
+                    </li>
+                    <li class="nav-item mx-2">
+                        <a class="nav-link" href="#">News</a>
+                    </li>
+                    <li class="nav-item mx-2 border-nav">
+                        <a class="nav-link" href="#">Empowerment Indigenise</a>
+                    </li>
+
+                </ul>
+
+            </div>
+
+        </div>
+
+    </nav>
+
+    <div class="container-fluid intro-bg pb-5">
+        <section class="intro container pt-5 first-page">
+            <p class="phello-text">"Hello! Kadoma community"</p>
+            <h1>KADOMA COMMUNITY<br> DEVELOPMENT COLLABORATIONS AND<br> WORK ALONGS</h1>
+            <div class="intro-d">
+                <p class="intro-details my-3 py-3">
+                    changing communities and wards through self development projects to empower the future generation wealth 
+                    to the communities and enriching lives through employment creation
+                </p>
+                <h2 class="date my-5">2021.10.27(MON)~30(WED)</h2>
+                <p class="location"> @Rimuka Stadium Ward 34 Council</p>
+            </div>
```

Comment:
- [ ] Please ensure the section in the attached image looks like the given design on mobile screen size.

| Yours | Original |
| ------ | ------ |
| <img src="https://user-images.githubusercontent.com/52497246/133553661-1092a4cd-d808-4bba-803f-483bf6ad9c33.png" height="250px"/> | <img src="https://user-images.githubusercontent.com/33323077/133798572-765985e7-3806-4c61-8503-db265ef5213c.png" height="250px"/>